apps/web/store/functions.ts

```diff
@@ -190,8 +192,10 @@ export function renameGraph(graphId: string, title: string) {
  */
 export function setBuildTarget(degreeId: string) {
   trpc
-    .query('degree', degreeId)
-    .then((degree) => trpc.query('modules', degree.modules))
+    .query('degree', { degreeId })
+    .then((degree) =>
+      trpc.query('modules', { moduleCodes: degree.modules.join(',') })
+    )
```

Comment:
downside of parsing in the `resolve()`: frontend must pass in a comma separated string, rather than an array